suite-native/module-settings/src/screens/SettingsPrivacyAndSecurity.tsx

```diff
@@ -111,6 +112,11 @@ export const SettingsPrivacyAndSecurity = () => {
 
     return (
         <Screen
+            screenHeader={
+                <ScreenHeader>
```

Comment:
Why it's <ScreenHeader> not used directly in <Screen>?

Reply:
@PeKne Good point, but the reason for doing this is that there is no place to do it. I would need to create a new package for that due to circular deps. That's IMO overkill as this is more flexible if we need to change it on some screens (as it's already introduced in new designs)

@Nodonisko That could work if we only had one type of screen header. And in the end, we might migrate to the solution, because the SubHeader will be deleted and we will have 2 types of headers. Right now, if I put ScreenHeader into screen, it won't work for the header, that's on connect device screens.

Reply:
https://github.com/trezor/trezor-suite/pull/9749/commits/5ea6ff829539e29fdf1dbffd55a8c192681fa104